src/geometry/manifold/complex.lean

```diff
@@ -5,7 +5,8 @@ Authors: Heather Macbeth
 -/
 import analysis.complex.abs_max
 import analysis.locally_convex.with_seminorms
-import geometry.manifold.mfderiv
+import geometry.manifold.cont_mdiff_mfderiv
+import geometry.manifold.sheaf.smooth
```

Comment:
it feels strange to me to add new imports just for a sanity check. Maybe move this to a new file?

Reply:
I had the idea that eventually there will be lots of material in `geometry.manifold.complex` using sheaves.  However, it's true that this one example use is not very fundamental.  If you like I can just remove it from the PR?

Reply:
I think it would be better if the sheaf import would forever be banned from the basic file `geometry.manifold.complex`. So creating a new file with the example looks good to me.